src/components/ProductFilterModal.vue

```diff
@@ -84,26 +93,46 @@ export default defineComponent({
       modalController.dismiss({ dismissed: true, isFilterChanged: this.isFilterChanged });
     },
     async search(event: any) {
-      // TODO: need to implement infinite scroll on the modal search
+      this.queryString = event.target.value;
+      this.facetOptions = [];
+      this.getTags();
+    },
+    async getTags(vSize?: any, vIndex?: any) {
```

Comment:
I think we don't need separate methods for search and getTags, we can use a single method.

Reply:
yes sir, actually I followed SelectProduct component method style, which uses searchProduct on keypress and loadMoreProducts on infinite scroll